Swift/Tests/PublisherTest.swift

```diff
@@ -22,7 +22,7 @@ import XCTest
 @testable import CouchbaseLiteSwift
 
 @available(iOS 13.0, *)
-class PublisherTest: CBLTestCase {
+class PublisherTest: QueryTest {
```

Comment:
Does this also run all QueryTest tests as well? If that happens, alternatively, you could move `createDocNumbered` to the base test class.

Reply:
no it doesnt, but good catch. That is a leftover from something I've tried previously. It is not needed - `createDocNumbered` is in CBLTestCase. I will change it back